README.md

```diff
@@ -32,4 +34,4 @@ We would love to hear your feedback. Submit [an issue](https://github.com/golang
 
 ## License
 
-GNU GPLv3 &copy; The [golang.design](https://golang.design) Authors. Originally created by [Changkun Ou](https://changkun.de).
\ No newline at end of file
+GNU GPLv3 &copy; The [golang.design](https://golang.design) Authors. Originally created by [Changkun Ou](https://changkun.de).
```

Comment:
the line is identical. I think some non-visible symbol is different. I commit form macOS, maybe you did windows and you have different new line symbol? anws. this line is the same!!